City-of-Helsinki/drupal-hdbt-admin
# UHF-9690: Character counter

# [UHF-9690](https://helsinkisolutionoffice.atlassian.net/browse/UHF-9690)


## What was done


* Moved the logic to field widget and themed the character counter accordingly


## How to install and test
* Check instructions from platform config: https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/704

## Other PRs


* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/704


[UHF-9690]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-9690?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
